code/modules/mob/living/carbon/xenomorph/castes/widow/abilities_widow.dm

```diff
@@ -95,7 +95,7 @@
 			continue
 		if(HAS_TRAIT(victim, TRAIT_LEASHED))
 			continue
-		if(!check_path(src, victim, projectile = TRUE))
+		if(check_path(src, victim, pass_flags_checked = PASS_PROJECTILE) != get_turf(victim))
 			continue
```

Comment:
doesnt check path return a list?

Reply:
no, it returns the last turf it can reach in the path (I could make it return the whole list if you think theres a use for it though)